modules/ROOT/pages/salesforce/salesforce-mktg-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,112 @@
+= Salesforce Marketing Cloud - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Upgrade Anypoint Connector for Salesforce Marketing Cloud (Salesforce Marketing Cloud Connector) to version 3.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|2.2.x |3.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Removed support for FuelSDK. The connector now uses the Mule HTTP client to perform the SOAP requests to the API.
+* Changed the supported connection types
+* Changed the Upsert entities operation
+* Removed some operations and grouped their functionality into fewer operations
+
+== Changes in Connection Types
+
+This release contains the following changes to connection types:
+
+* Added support for using OAuth authentication
+* Removed OAuth Username and Password authentication, because the Salesforce Marketing Cloud deprecated this authentication type
```

Comment:
```suggestion
* Removed OAuth username and password authentication, because Salesforce Marketing Cloud deprecated this authentication type
```
Or...should we be calling these authentication types? And which did SMC deprecate: both? types?